Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -454,6 +456,8 @@ GDCMImageIO::InternalReadImageInformation()
 
   // In general this should be relatively safe to assume
   gdcm::ImageHelper::SetForceRescaleInterceptSlope(true);
+  // Secondary capture image orientation patient and image position patient support
+  gdcm::ImageHelper::SetSecondaryCaptureImagePlaneModule(true);
```

Comment:
@thewtex Maybe something like
```
#if (defined(ITK_USE_SYSTEM_GDCM) || ((GDCM_MAJOR_VERSION == 3 && GDCM_MINOR_VERSION == 0 && GDCM_BUILD_VERSION > 23) || GDCM_MAJOR_VERSION > 3))
```
```
#endif
```
around _SetSecondaryCaptureImagePlaneModule_? Otherwise, it may cause build failures using external GDCM. There is currently no version of GDCM with this feature.

Reply:
@issakomi good idea, added here: https://github.com/InsightSoftwareConsortium/ITK/pull/4601